.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,343 @@
+### Table of Contents
+1. [Overview](#Overview)<br />
+2. [Minimum Recommended Skillset](#MinSkillset)<br />
+3. [Required Reading](#ReqReading)<br />
+4. [Development Practices](#DevelopmentPractices)<br />
+4.1. [Share Early, Share Often](#ShareEarly)<br />
+4.2. [Common Mistakes](#CommonMistakes)<br />
+4.3. [Testing](#Testing)<br />
+4.4. [Code Documentation and Commenting](#CodeDocumentation)<br />
+4.5. [Model Git Commit Messages](#ModelGitCommitMessages)<br />
+5. [Code Approval Process](#CodeApproval)<br />
+5.1 [Code Review](#CodeReview)<br />
+5.2 [Rework Code (if needed)](#CodeRework)<br />
+5.3 [Acceptance](#CodeAcceptance)<br />
+6. [FAQs](#FrequentlyAskedQuestions)<br />
+6.1. [Balances](#Balances)<br />
+6.2. [Confirmations](#Confirmations)<br />
+7. [Contribution Standards](#Standards)<br />
+7.1. [Contribution Checklist](#Checklist)<br />
+7.2. [Licensing of Contributions](#Licensing)<br />
+
+<a name="Overview" />
+
+### 1. Overview
+
```

Comment:
Great overview explaining how its important to make sure that regressions do not happen

Reply:
Thanks!